### PR TITLE
Dont try and tag branches in docker if its a pull request build

### PR DESCRIPTION
**Pull request by tdfischer**

### CODE REVIEW

**Comment by tdfischer** [comment]
All fixed up. Ready for review+merge.

**Comment by nbougalis** [comment]
Thank you!

**Comment by ximinez** [comment]
Suggested commit message:
"circle: docker: Don't tag branches if its a PR (PKG-33)"

Prevents github from wrapping message (56 chars), and puts ticket number on first line.

File: Builds/Docker/build-ci.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+set -e
+
+mkdir -p build/docker/
+cp doc/rippled-example.cfg build/clang.debug/rippled build/docker/
+cp Builds/Docker/Dockerfile-testnet build/docker/Dockerfile
+mv build/docker/rippled-example.cfg build/docker/rippled.cfg
+strip build/docker/rippled
+docker build -t ripple/rippled:$CIRCLE_SHA1 build/docker/
+docker tag ripple/rippled:$CIRCLE_SHA1 ripple/rippled:latest
```
Comment (ximinez) [inline Builds/Docker/build-ci.sh]: Should this tag also be in the `if`?

**Comment by vinniefalco** [comment]
Commit message needs work

File: Builds/Docker/build-ci.sh (hunk 1 above)
Comment (tdfischer) [inline Builds/Docker/build-ci.sh]: No, the underlying bug was the $CIRCLE_BRANCH variable resolving to 'pull/8375', which isn't a valid tag for docker. $CIRCLE_SHA1 though is always a sha1 hash which is fine.

**Comment by tdfischer** [comment]
updated message

File: Builds/Docker/build-ci.sh (hunk 1 above)
Comment (ximinez) [inline Builds/Docker/build-ci.sh]: Makes sense.

**Comment by ximinez** [comment]
:+1:

**Comment by nbougalis** [comment]
Ed: if you're happy with this, please mark as Passed.

**Comment by ximinez** [comment]
Vinnie beat me to it, but I'm good.

**Comment by tdfischer** [comment]
Was this merged somewhere? Github says it was just closed.

**Comment by ximinez** [comment]
Merged as https://github.com/ripple/rippled/commit/172a0603307dfd17642ae3d90aa9afedb919bc44

**Comment by vinniefalco** [comment]
Its like Ragu - "it's in there"

**Pull request closed without merging**